src/common/lib/utils.ts

```diff
@@ -45,7 +45,7 @@ const utils = {
   openUrl: (url: string) => {
     browser.tabs.create({ url });
   },
-  openPrompt: <Type>(message: {
+  openPrompt: async <Type>(message: {
```

Comment:
if you return a promise you don't need this async. So I think it's using async OR returning a Promise.

Reply:
Some unrelated changes seemed to slip into this branch. Removed this stuff again. (also please don't review draft PRs if not explicitly requested)